### PR TITLE
Export clearTags and stops escaping &"<>

### LINKED ISSUE #26: filterTags escapes HTML entities because of TagSoup's defaults
``` Haskell
*Text.HTML.SanitizeXSS> filterTags safeTags "text&nbsp;more text"
"text&amp;nbsp;more text"
```
This would display as "text &nbspmore text" instead of "text more text".

If you add `optEscape = id` to the `renderOptions` then TagSoup will stop trying to escape &"<>

``` Haskell
*Text.HTML.SanitizeXSS> filterTags safeTags "text&nbsp;more text"
"text&nbsp;more text"
```

If you are ok with this fix, I'll create a pull request with my changes.

### PR DESCRIPTION
**Pull request by Skyfold**
Fixes #26 #25

### CODE REVIEW

**Comment by snoyberg** [comment]
It would be great to include some test cases to cover the issues this PR is fixing.

**Comment by Skyfold** [comment]
Sure, I'll add some this weekend.

File: flake.lock [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
```
Comment (snoyberg) [inline flake.lock:1]: I'd prefer not including Nix-specific files, I won't be able to maintain them.

**Review by snoyberg — CHANGES_REQUESTED** [review]

File: flake.lock (hunk 1 above)
Comment (Skyfold) [reply flake.lock:1]: I removed the `flake.nix`

**Review by snoyberg — APPROVED** [review]

**Pull request merged**